MobileCenterUpdates/MobileCenterUpdates/MSUpdatesPrivate.h

```diff
@@ -47,4 +62,31 @@ static NSString *const kMSUpdtsDefaultCustomScheme = @"msupdt";
  */
 - (void)openURLInSafariApp:(NSURL *)url;
 
+/**
+ * Take a request via custom URL scheme from browser.
+ *
+ * @param url  The url with parameters.
+ */
+- (void)openUrl:(NSURL *)url;
```

Comment:
I would rephrase it like this : "Process URL related to this module". May be triggered not only by browsers in the future.

Also may be add a discussion saying that default integration is to place this call into the app delegate openUrl method.